Added automatic seeding of all available seeds.

nenti opened the pull request.

nenti committed:
Added automatic seeding of all available seeds.

AshanFernando commented:
@nenti Thanks for the PR, can you also update the Readme for this?

mattfysh commented on index.js:
```diff
@@ -181,7 +181,7 @@ class ServerlessDynamodbLocal {
         const config = this.service.custom.dynamodb;
         const seedConfig = _.get(config, "seed", {});
-        const seed = this.options.seed;
+        const seed = this.options.seed ? this.options.seed : Object.keys(seedConfig).join(',');
```
@nenti - this is awesome, in JS you could rep this as

```javascript
const seed = this.options.seed || Object.keys(seedConfig).join(',');
```

mattfysh commented:
+1 would love to see this on npm

rehrumesh committed:
Update index.js

rehrumesh commented:
@nenti would love to merge this soon after the readme update.

nenti committed:
Merge remote-tracking branch 'origin/v1' into v1

nenti committed:
Tweaked automatic seeding and readme.
Changed merge order for proper use of cli-commands e.g. --seed.

nenti commented:
@rehrumesh I added Readme changes. Aswell as some tweaks to the auto seeding. You can now use --seed command to overwrite your configuration. And with the old implementation using --seed without any seeds configures the plugin crashed. So I changed the seed handler for more robustness. 

In the reedme I added an example seed because this is something that I had to figure out and would have loved to be available there. Although I think we could label it to make sure the user understands what it is. I'm not a pro in markdown.

rehrumesh commented:
@nenti Thanks for the awesomeness. 🥇

amitaymolko commented:
Has this change been released to npm?

rehrumesh commented:
@amitaymolko @mattfysh @nenti NPM is updated with new release :)
